File: Java/Program/Manusia.java
public class Manusia {
    private String noktp;
    private String nama;
    private String alamat;

    public Manusia() {}

    public Manusia(String noktp, String nama, String alamat) {
        this.noktp = noktp;
        this.nama = nama;
        this.alamat = alamat;
    }

    public String getNoktp() {
        return noktp;
    }

    public void setNoktp(String noktp) {
        this.noktp = noktp;
    }

    public String getNama() {
        return nama;
    }

    public void setNama(String nama) {
        this.nama = nama;
    }

    public String getAlamat() {
        return alamat;
    }

    public void setAlamat(String alamat) {
        this.alamat = alamat;
    }

    public void tampilDataManusia() {
        System.out.println("No KTP   : " + getNoktp());
        System.out.println("Nama     : " + getNama());
        System.out.println("Alamat   : " + getAlamat());
    }
}

File: Python/Program/Manusia.py
class Manusia:
    def __init__(self, noktp="", nama="", alamat=""):
        self.__noktp = noktp
        self.__nama = nama
        self.__alamat = alamat

    # Getter dan setter NoKTP
    def get_noktp(self):
        return self.__noktp
    
    def set_noktp(self, noktp):
        self.__noktp = noktp

    # Getter dan Setter Nama
    def set_nama(self, nama):
        self.__nama = nama

    def get_nama(self):
        return self.__nama

    # Getter dan setter Alamat
    def set_alamat(self, alamat):
        self.__alamat = alamat

    def get_alamat(self):
        return self.__alamat

    # Method untuk menampilkan data
    def tampil_data_manusia(self):
        print("No KTP   :", self.get_noktp())
        print("Nama     :", self.get_nama())
        print("Alamat   :", self.get_alamat())